tests/logger_tests.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <spdlog/sinks/basic_file_sink.h>
+#include <spdlog/spdlog.h>
+#include <rmm/mr/device/cuda_memory_resource.hpp>
+#include <rmm/mr/device/logging_resource_adaptor.hpp>
+
+#include <gtest/gtest.h>
+
+TEST(Adaptor, first) {
+  rmm::mr::cuda_memory_resource upstream;
+
+  rmm::mr::logging_resource_adaptor<rmm::mr::cuda_memory_resource> log_mr{
+      &upstream, "logs/test1.txt"};
+
+  auto p = log_mr.allocate(100);
+  log_mr.deallocate(p, 100);
+}
+
+TEST(Adaptor, factory) {
```

Comment:
So, these tests aren't requiring anything. In fact there's not even EXPECT_NO_THROW... I think they should have some way of failing. Ideally should read in the log file and verify it has the expected number of rows and the expected sizes?

Reply:
Yeah, these tests were just to make sure it compiled